integrationtest/src/it/scala/io/gearpump/integrationtest/checklist/RestServiceSpec.scala

```diff
@@ -294,7 +294,27 @@ class RestServiceSpec extends TestSpecBase {
 
   "application life-cycle" should {
     "newly started application should be configured same as the previous one, after restart" in {
+      // setup
+      val originAppId = restClient.submitApp(wordCountJar)
+      val originAppsCount = restClient.listRunningApps().length
+      val originAppDetail = restClient.queryStreamingAppDetail(originAppId)
+      val originProcessors = originAppDetail.processors
```

Comment:
You should keep `originAppDetail` only. There is no need to store originProcessor, originSplitNum, etc.